osrsbox/items_api/stance.py

```diff
@@ -0,0 +1,43 @@
+from dataclasses import dataclass
+from enum import Enum
+from typing import Optional
+
+
+class AttackType(Enum):
+    STAB = "stab"
+    MAGIC = "magic"
+    DEFENSIVE_CASTING = "defensive casting"
+    SLASH = "slash"
+    RANGED = "ranged"
+    SPELLCASTING = "spellcasting"
+    CRUSH = "crush"
+    NONE = None
+    ALIAS_FOR_NONE = "none"
+
+
+class AttackStyle(Enum):
+    ACCURATE = "accurate"
+    AGGRESSIVE = "aggressive"
+    MAGIC = "magic"
+    DEFENSIVE = "defensive"
+    CONTROLLED = "controlled"
+    NONE = None
+    ALIAS_FOR_NONE = "none"
+
+
+@dataclass
+class Stance:
+    boosts: Optional[str]
+    combat_style: str
+    experience: str
```

Comment:
It looks like `experience` should be an `Optional[str]` - the `Blasters` weapon type is the only weapon type to not have any experience associated with it.